### PR TITLE
Model Submission Coordinated Robotics Mike

### PR DESCRIPTION
**Pull request by knoedler**
This is a 4 wheel skid steer robot using bicycle wheels.  A pivot allows the left and right halves to rotate and hopefully deal better with terrain.  It again uses the same sensor platform as Karen.  Validation is not yet completed so validated items are set to the default 1m/s and 1 hour.

### CODE REVIEW

File: submitted_models/coro_mike_sensor_config_1/specifications.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Cost breakdown:
+$2500 - Velodyne VLP-16 https://www.ebay.com/sch/i.html?_nkw=velodyne+vlp-16&_sacat=0
+$1800 - Frame / motors / wheels / batteries / controllers
+$1300 - MS-IMU3025
+$800  - lights / connectors /switches / DC-DC / misc
+$800  - Computer
+$225  - Seek Thermal Compact
+$200  - D435
+
+## Sensors
```
Comment (acschang) [inline submitted_models/coro_mike_sensor_config_1/specifications.md:31]: Please include the Seek Thermal camera in this section as well.

Comment (knoedler) [reply submitted_models/coro_mike_sensor_config_1/specifications.md:31]: I added the thermal sensor to the list of sensors.

**Review by acschang — APPROVED** [review]
The initial assessment of the model is complete and no changes are currently warranted.  This submission will continue to be evaluated in the next stage of our review process.

**Comment by caguero** [comment]
I was able to load the robot, visualize all sensors in RViz, and control it in velocity. The battery discharges for me as well and I don't see anything wrong here. Ready to approve.
mike_the_robot

**Review by caguero — APPROVED** [review]

**Comment by nkoenig** [comment]
Size[1.4342 0.85 0.924098]

**Pull request merged**